This clearly doesn't belong in officecfg ... but where does it belong ?


--- officecfg/util/galeon-helper	Thu Apr 11 09:25:15 2002
+++ officecfg/util/galeon-helper	Fri Dec 20 14:00:33 2002
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+unset LD_LIBRARY_PATH
+unset MOZILLA_FIVE_HOME
+gnome-url-show "$@"

Index: officecfg/prj/d.lst
===================================================================
RCS file: /cvs/util/officecfg/prj/d.lst,v
retrieving revision 1.32
diff -u -p -u -r1.32 d.lst
--- officecfg/prj/d.lst	26 Mar 2003 13:49:49 -0000	1.32
+++ officecfg/prj/d.lst	20 May 2003 11:00:38 -0000
@@ -9,6 +9,8 @@ mkdir: %_DEST%\xml%_EXT%\registry\schema
 ..\%__SRC%\misc\registry\schema\org\openoffice\Office\*.xcs %_DEST%\xml%_EXT%\registry\schema\org\openoffice\Office
 ..\%__SRC%\misc\registry\schema\org\openoffice\ucb\*.xcs %_DEST%\xml%_EXT%\registry\schema\org\openoffice\ucb
 
+..\util\galeon-helper %_DEST%\bin%_EXT%\galeon-helper
+
 mkdir: %_DEST%\xml%_EXT%\registry\data
 mkdir: %_DEST%\xml%_EXT%\registry\data\org
 mkdir: %_DEST%\xml%_EXT%\registry\data\org\openoffice
Index: officecfg/util/makefile.mk
===================================================================
RCS file: /cvs/util/officecfg/util/makefile.mk,v
retrieving revision 1.1
diff -u -p -u -r1.1 makefile.mk
--- officecfg/util/makefile.mk	16 May 2002 17:50:31 -0000	1.1
+++ officecfg/util/makefile.mk	20 May 2003 11:00:47 -0000
@@ -79,4 +79,8 @@ $(BIN)$/registry{$(alllangext)}.zip : $(
 
 ALLTAR: \
 	$(MISC)$/$(TARGET)_delzip \
-	$(BIN)$/registry{$(alllangext)}.zip
\ No newline at end of file
+	$(BIN)$/registry{$(alllangext)}.zip \
+	$(BIN)$/galeon-helper
+
+$(BIN)$/galeon-helper:
+	cp galeon-helper $(BIN)$/galeon-helper
Index: scp/source/office/files.scp
===================================================================
RCS file: /cvs/installation/scp/source/office/files.scp,v
retrieving revision 1.344
diff -u -p -u -r1.344 files.scp
--- scp/source/office/files.scp	24 Apr 2003 11:14:15 -0000	1.344
+++ scp/source/office/files.scp	20 May 2003 11:02:06 -0000
@@ -2505,6 +2482,13 @@ End
 
 #ifdef UNX                              // ACHUNG: Hier reine Unix-Dateien (Solaris und Linux)
 
+File GID_FILE_BIN_GALEONHELPER
+	BIN_FILE_BODY;
+	Dir		= GID_DIR_PROGRAM;
+	Name		= "galeon-helper";
+	Styles		= (PACKED);
+End
+
 File GID_FILE_BIN_PLUGINAPP
         BIN_FILE_BODY;
         Dir             = GID_DIR_PROGRAM;
Index: sfx2/source/dialog/mailmodel.cxx
===================================================================
RCS file: /cvs/framework/sfx2/source/dialog/mailmodel.cxx,v
retrieving revision 1.21
diff -u -p -u -r1.21 mailmodel.cxx
--- sfx2/source/dialog/mailmodel.cxx	17 Apr 2003 16:09:52 -0000	1.21
+++ sfx2/source/dialog/mailmodel.cxx	20 May 2003 11:11:45 -0000
@@ -784,7 +784,14 @@ SfxMailModel_Impl::SendMailResult SfxMai
 						}
 
 						Sequence< OUString > aAttachmentSeq( 1 );
+#ifdef UNX
+						// This is a nasty hack - as anyone can see; we should accept URIs
+						// instead of plain filenames in evolution instead of hobbling other people.
+						INetURLObject url (aFileName);
+						aAttachmentSeq[0] = url.getFSysPath (INetURLObject::FSYS_UNX);
+#else
 						aAttachmentSeq[0] = aFileName;
+#endif
 
 						xSimpleMailMessage->setSubject( maSubject );
 						xSimpleMailMessage->setAttachement( aAttachmentSeq );
Index: officecfg/registry/data/org/openoffice/Office/Common.xcu
===================================================================
RCS file: /cvs/util/officecfg/registry/data/org/openoffice/Office/Common.xcu,v
retrieving revision 1.13
diff -u -p -u -r1.13 Common.xcu
--- officecfg/registry/data/org/openoffice/Office/Common.xcu	2 Jul 2003 14:19:36 -0000	1.13
+++ officecfg/registry/data/org/openoffice/Office/Common.xcu	29 Aug 2003 13:48:20 -0000
@@ -107,19 +107,19 @@
   </node>
   <node oor:name="ExternalApps">
     <prop oor:name="http" oor:op="replace" oor:type="xs:string">
-      <value/>
+      <value>galeon-helper</value>
     </prop>
     <prop oor:name="https" oor:op="replace" oor:type="xs:string">
-      <value/>
+      <value>galeon-helper</value>
     </prop>
     <prop oor:name="ftp" oor:op="replace" oor:type="xs:string">
-      <value/>
+      <value>galeon-helper</value>
     </prop>
     <prop oor:name="file" oor:op="replace" oor:type="xs:string">
-      <value/>
+      <value>nautilus</value>
     </prop>
     <prop oor:name="mailto" oor:op="replace" oor:type="xs:string">
-      <value/>
+      <value>evolution-1.4</value>
     </prop>
   </node>
   <node oor:name="ExternalMailer">
Index: officecfg/registry/schema/org/openoffice/Office/Common.xcs
===================================================================
RCS file: /cvs/util/officecfg/registry/schema/org/openoffice/Office/Common.xcs,v
retrieving revision 1.84
diff -u -p -u -r1.84 Common.xcs
--- officecfg/registry/schema/org/openoffice/Office/Common.xcs	24 Jul 2003 13:12:08 -0000	1.84
+++ officecfg/registry/schema/org/openoffice/Office/Common.xcs	29 Aug 2003 13:52:15 -0000
@@ -9499,9 +9499,9 @@
 						<label xml:lang="th">AutoSave every</label>
 						<label xml:lang="x-comment">AutoSave every</label>
 					</info>
-					<value>false</value>
+					<value>true</value>
 				</prop>
-				<prop oor:name="CreateBackup" oor:type="xs:boolean">
+				<prop oor:name="CreateBackup" oor:type="xs:boolean"> <!-- dummy -->
 					<!-- OldPath: General/Save/Documents -->
 					<!-- OldLocation: soffice.cfg -->
 					<!-- UIHints: Tools  Options - General  Save - [Section] Save -->
@@ -26874,9 +26874,9 @@
 					<desc xml:lang="th">Specifies the external mail application to be used.</desc>
 					<desc xml:lang="x-comment">Specifies the external mail application to be used.</desc>
 				</info>
-				<value/>
+				<value>evolution-1.4</value>
 			</prop>
-			<prop oor:name="CommandProfile" oor:type="xs:string">
+			<prop oor:name="CommandProfile" oor:type="xs:string"> <!-- dummy -->
 				<info>
 					<desc xml:lang="en-US">Specifies the default external mail application.</desc>
 					<desc xml:lang="el">Specifies the default external mail application.</desc>
@@ -26905,9 +26905,9 @@
 					<desc xml:lang="th">Specifies the default external mail application.</desc>
 					<desc xml:lang="x-comment">Specifies the default external mail application.</desc>
 				</info>
-				<value>Mozilla 1.0 - 1.2</value>
+				<value>Evolution 1.4 or later</value>
 			</prop>
-			<set oor:name="Profiles" oor:node-type="MailCommandLineProfile">
+			<set oor:name="Profiles" oor:node-type="MailCommandLineProfile"> <!-- dummy -->
 				<info>
 					<desc xml:lang="en-US">Contains a set of supported command line profiles.</desc>
 					<desc xml:lang="el">Contains a set of supported command line profiles.</desc>
